boards/nucleo-f103rb/include/periph_conf.h

```diff
@@ -147,35 +147,37 @@ static const uart_conf_t uart_config[] = {
  * @name I2C configuration
  * @{
  */
-#define I2C_NUMOF           (2U)
-#define I2C_0_EN            1
-#define I2C_1_EN            0
-#define I2C_IRQ_PRIO        1
-#define I2C_APBCLK          (CLOCK_APB1)
-
-/* I2C 0 device configuration */
-#define I2C_0_DEV           I2C1
-#define I2C_0_CLKEN()       (periph_clk_en(APB1, RCC_APB1ENR_I2C1EN))
-#define I2C_0_CLKDIS()      (periph_clk_dis(APB1, RCC_APB1ENR_I2C1EN))
-#define I2C_0_EVT_IRQ       I2C1_EV_IRQn
-#define I2C_0_EVT_ISR       isr_i2c1_ev
-#define I2C_0_ERR_IRQ       I2C1_ER_IRQn
-#define I2C_0_ERR_ISR       isr_i2c1_er
-/* I2C 0 pin configuration */
-#define I2C_0_SCL_PIN       GPIO_PIN(PORT_B, 8) /* remapped */
-#define I2C_0_SDA_PIN       GPIO_PIN(PORT_B, 9) /* remapped */
-
-/* I2C 1 device configuration */
-#define I2C_1_DEV           I2C2
-#define I2C_1_CLKEN()       (periph_clk_en(APB1, RCC_APB1ENR_I2C2EN))
-#define I2C_1_CLKDIS()      (periph_clk_dis(APB1, RCC_APB1ENR_I2C2EN))
-#define I2C_1_EVT_IRQ       I2C2_EV_IRQn
-#define I2C_1_EVT_ISR       isr_i2c2_ev
-#define I2C_1_ERR_IRQ       I2C2_ER_IRQn
-#define I2C_1_ERR_ISR       isr_i2c2_er
-/* I2C 1 pin configuration */
-#define I2C_1_SCL_PIN       GPIO_PIN(PORT_B, 10)
-#define I2C_1_SDA_PIN       GPIO_PIN(PORT_B, 11)
+static const i2c_conf_t i2c_config[] = {
+    {
+        .dev            = I2C1,
+        .speed          = I2C_SPEED_NORMAL,
+        .scl_pin        = GPIO_PIN(PORT_B, 8),
+        .sda_pin        = GPIO_PIN(PORT_B, 9),
+        .scl_af         = GPIO_AF_OUT_OD,
+        .sda_af         = GPIO_AF_OUT_OD,
```

Comment:
here and in many other places related to stm32f1, the af attributes should be removed. Is it possible to just rebase the branch and remove the related commits. I saw the right fix in another PR, the corresponding commit should be applied then.

Reply:
This has been fix on (new_i2c_if branch)[https://github.com/RIOT-OS/RIOT/blob/new_i2c_if/boards/nucleo-f103/include/periph_conf.h]
Looks like these values are re-introduced by the rebase because nucleo were renamed..
I'll try to find why again.